bundle-workflow/src/read_write_s3.py

```diff
@@ -0,0 +1,71 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import os
+
+import boto3
+
+
+class ReadWriteFiles:
```

Comment:
This should represent a thing, e.g. `S3` or `S3Bucket` and take bucket as a parameter since it's required in both put/get.